pkg/receive/handler.go

```diff
@@ -74,48 +65,69 @@ func instrumentHandler(handlerName string, handler http.HandlerFunc) http.Handle
 	)
 }
 
+func NewHandlerInMux(mux *http.ServeMux, logger log.Logger, o *Options) *Handler {
```

Comment:
I think you forgot to remove this or you don't agree with my comment?

Reply:
No, just overlooked it, sorry.